orchestrator/careplancontributor/handle_taskfiller_test.go

```diff
@@ -344,7 +347,7 @@ func TestService_handleTaskFillerCreate(t *testing.T) {
 					*result = primaryTask
 					return nil
 				}).AnyTimes()
-
+			notifierMock.EXPECT().NotifyTaskAccepted(mockFHIRClient, gomock.Any()).AnyTimes()
```

Comment:
might want to actually test the behavior in this test

Reply:
Good one, fixed.